fabric/main.py

```diff
@@ -589,7 +589,12 @@ def update_output_levels(show, hide):
 
 
 def show_commands(docstring, format, code=0):
-    print("\n".join(list_commands(docstring, format)))
+    content = "\n".join(list_commands(docstring, format))
+    if isinstance(content, unicode):
+        import codecs, locale
```

Comment:
I don't think it helps to import these standard library modules conditionally. Why not import them at the top of the file like usual?